spring-core/src/main/java/org/springframework/asm/ByteVector.java

```diff
@@ -354,7 +354,7 @@ public ByteVector putByteArray(
   private void enlarge(final int size) {
     int doubleCapacity = 2 * data.length;
     int minimalCapacity = length + size;
-    byte[] newData = new byte[doubleCapacity > minimalCapacity ? doubleCapacity : minimalCapacity];
+    byte[] newData = new byte[Math.max(doubleCapacity, minimalCapacity)];
```

Comment:
Please refrain from modifying classes under `org.springframework.asm`, `org.springframework.cglib`, and `org.springframework.objenesis`. Those include repackaged forks of the third-party libraries ASM, CGLIB, and Objenesis. Any refactoring to those classes should take place upstream in the originating repository. The Spring Framework will then pick up the changes when syncing with official updates of the forked third-party libraries.

----

Note, however, that there is no need to update this PR. I'll simply revert that before merging.